tensorflow/lite/micro/examples/micro_speech/esp/Makefile.inc

```diff
@@ -0,0 +1,40 @@
+# Adding some esp specific files in the main CMakeLists.txt
+ESP_MICRO_SPEECH_SRCS := \
+tensorflow/lite/micro/examples/micro_speech/esp/ringbuf.c \
```

Comment:
When defining variables across multiple lines, the last line should omit the trailing `\` (backslash).

Reply:
Thanks for the update.

> I am afraid There is one more thing required here i.e. a way to copy `sdkconfig.defaults` from `examples/micro_speech/esp` to the home folder of the project (`micro_speech/esp-idf/` ).

I updated the PR with a new commit: https://github.com/AdityaHPatwardhan/tensorflow/pull/1/commits/3b429c79fb5862e51a26bd87ebc0d34c2815b3ba

Does this fix the issue ?

Reply:
Oops, I did not notice the `Kconfig.projbuild` requirement. Let's handle that in the person detector PR instead then.

Reply:
> Does this fix the issue ?

Yes , Thanks for the build fixes , I have merged your PR into my branch.